apps/speed.c

```diff
@@ -933,11 +1459,28 @@ int speed_main(int argc, char **argv)
         goto end;
     }
 
+    if (async_jobs > 0) {
+        if (!ASYNC_init(1, 0, 0)) {
```

Comment:
Yes, we should allow that. Probably the condition should be `max_size > 0 && init_size > max_size`